### ISSUE TITLE
Possible bug in GFED4 Hg0 Emission Factors

### ISSUE THREAD

**Issue by emroy00** [geoschem/geos-chem#1817]
### Name and Institution (Required)

Name: Eric Roy
Institution: MIT

### Confirm you have reviewed the following documentation

- [x] [Support guidelines](https://geos-chem.readthedocs.io/en/stable/reference/SUPPORT.html)
- [x] [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [x] [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [x] [Current and past Github issues](https://github.com/geoschem/geos-chem/issues)

### Description of your issue or question

We may have found a bug in the GFED4s Hg0 Biomass Burning emission factors:

https://github.com/geoschem/HEMCO/blob/4e818813da35bf76dbcee2da6ecc598b9c89b1a7/src/Extensions/hcox_gfed_include_gfed4.H#L328-L335

Past GC Hg0 biomass burning emission factors (EFs) have been calculated as the product of a constant Hg0:CO ratio and the CO EFs for the respective land use category. The constant Hg0:CO ratio suggested by Holmes et al. (2010) was 100 nmol mol<sup>-1</sup> (or 7.17e-7 g g<sup>-1</sup>), however current EFs exhibit Hg0:CO ratios that vary by land use category (courtesy of @arifein):
<img width="1111" alt="image" src="https://github.com/geoschem/geos-chem/assets/122998903/822b19c3-aeb1-408d-b50a-717a96968a65">

 @arifein found that the change was implemented in 2018 during an update of GFED4 emission factors:

https://github.com/geoschem/geos-chem/commit/131cb5da0708966c7ec695f83b64a8c28aa074b8#diff-fadca1025a23200fb81c647a278d4100d5afb14632b3b40c68e17f65e32321e4

### Proposed Fix

To ensure that EFs match existing literature and to highlight that these factors scale with CO, we suggest the changes to lines 328-335 of [HEMCO](https://github.com/geoschem/HEMCO/tree/4e818813da35bf76dbcee2da6ecc598b9c89b1a7)/[src](https://github.com/geoschem/HEMCO/tree/4e818813da35bf76dbcee2da6ecc598b9c89b1a7/src)/[Extensions](https://github.com/geoschem/HEMCO/tree/4e818813da35bf76dbcee2da6ecc598b9c89b1a7/src/Extensions)/hcox_gfed_include_gfed4.H:
```F90
Inst%GFED4_EMFAC(32,1)=7.17E-07_hp * Inst%GFED4_EMFAC(1,1) ! SAV -- SAVA
Inst%GFED4_EMFAC(32,2)=7.17E-07_hp * Inst%GFED4_EMFAC(1,2) ! FOR -- BORF 
Inst%GFED4_EMFAC(32,3)=7.17E-07_hp * Inst%GFED4_EMFAC(1,3) ! FOR -- TEMP
Inst%GFED4_EMFAC(32,4)=7.17E-07_hp * Inst%GFED4_EMFAC(1,4) ! DEF -- DEFO
Inst%GFED4_EMFAC(32,5)=7.17E-07_hp * Inst%GFED4_EMFAC(1,5) ! PET -- PEAT
Inst%GFED4_EMFAC(32,6)=7.17E-07_hp * Inst%GFED4_EMFAC(1,6) ! AGW -- AGRI
```
Initial comparisons of the impact of this change on annual Hg0 biomass burning emissions suggest a minor effect, though ensuring EFs are consistent with existing literature is important for a strong basis. This impact is shown in the plots below for 2015:

![image](https://github.com/geoschem/geos-chem/assets/122998903/a31cd448-3f67-4ef5-ac9f-f31056be209a)

Also tagging collaborators @arifein and @jennyfisher!

**Comment by yantosca** [comment]
Thanks for writing @emroy00.  We should be able to add this into an upcoming version such as 14.2.1 or 14.3.0.  It seems like a simple fix.

**Comment by msulprizio** [comment]
@emroy00 Could you submit a pull request as is described in our [contributing guidelines](https://geos-chem.readthedocs.io/en/stable/reference/CONTRIBUTING.html#how-can-i-submit-updates)?

**Comment by msulprizio** [comment]
Thanks for the pull request @emroy00. As Bob said, we should be able to include this in 14.2.1.

**Comment by yantosca** [comment]
PR https://github.com/geoschem/HEMCO/pull/215 has now been merged into the GEOS-Chem 14.2.1 development stream.  We can now close this issue.
